### ISSUE TITLE
Alignak modules configuration

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#955]
I recently noticed that the alignak modules configuration are all merged together...

What does it mean?

If you configure two modules which have some common attributes, only one instance of these common attributes are provided to the modules!

On one hand, it would allow to configure only once some common parameters (eg. backend url and credentials) 👍 
On another hand, it would not allow to have different values for the same parameter (eg. backend polling period, ...) 👎

**Comment by mohierf** [comment]
One more information, the module configuration may be used by a daemon:
```
[2017-12-05 12:12:09 CET] INFO: [alignak.module.backend_arbiter] StatsD configuration: 10.0.3.4:8125, prefix: alignak-mos.modules, enabled: True
[2017-12-05 12:12:09 CET] INFO: [alignak.stats] Sending module/backend_arbiter daemon statistics to: 10.0.3.4:8125, prefix: alignak-mos.modules
[2017-12-05 12:12:09 CET] INFO: [alignak.stats] Trying to contact StatsD server...
[2017-12-05 12:12:09 CET] INFO: [alignak.stats] StatsD server contacted
[2017-12-05 12:12:09 CET] INFO: [alignak.stats] Alignak internal statistics are sent to StatsD.

```

catched in the arbiter log. This is a module configuration, not the arbiter one!

**Comment by mohierf** [comment]
Closed 2.0
